src/utils/sqliteu.c

```diff
@@ -14,33 +14,43 @@
 
 #include "../utils/log.h"
 
+#define SQLITE_EXEC_TRIES 10000
+
 int execute_sqlite_query(sqlite3 *db, char *statement) {
-  char *err = NULL;
-  int rc = sqlite3_exec(db, statement, 0, 0, &err);
+  int rc = sqlite3_exec(db, statement, 0, 0, NULL);
 
   if (rc != SQLITE_OK) {
-    log_error("Failed to execute statement %s", err);
-    sqlite3_free(err);
-
+    log_error("Failed to execute statement: %s", sqlite3_errmsg(db));
     return -1;
   }
 
   return 0;
 }
 
-int check_table_exists(sqlite3 *db, char *table_name) {
-  sqlite3_stmt *res;
+int prepare_find_table(sqlite3 *db, char *table_name, sqlite3_stmt *res) {
```

Comment:
We're not editing this string, so we might as well make it `const`.

```suggestion
int prepare_find_table(sqlite3 *db, const char *table_name, sqlite3_stmt *res) {
```